content-resources/src/main/java/flyway/myplaces/V1_0_15__name_localization.java

```diff
@@ -0,0 +1,64 @@
+package flyway.myplaces;
+
+import fi.nls.oskari.log.LogFactory;
+import fi.nls.oskari.log.Logger;
+import fi.nls.oskari.util.JSONHelper;
+import fi.nls.oskari.util.PropertyUtil;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.json.JSONObject;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Migrate fields JSONObject to JSONArray from user_layer table
+ */
+public class V1_0_15__name_localization extends BaseJavaMigration {
+    private static final Logger LOG = LogFactory.getLogger(V1_0_15__name_localization.class);
+    private static final List<String> LANGUAGES = Arrays.asList(PropertyUtil.getSupportedLanguages());
+    private static final String DFFAULT_LANGUAGE = PropertyUtil.getDefaultLanguage();
+
+    public void migrate(Context context) throws Exception {
+        Connection conn = context.getConnection();
+        createLocaleColumn(conn);
+        setLocales(conn);
+        LOG.info("Migrated myplaces layer names to locales:", LANGUAGES );
+    }
+    private void createLocaleColumn (Connection connection) throws Exception {
+        final String sql = "ALTER TABLE categories ADD COLUMN IF NOT EXISTS locale json DEFAULT '{}'";
+        final PreparedStatement statement = connection.prepareStatement(sql);
+        try {
+            statement.execute();
+        } finally {
+            statement.close();
+        }
```

Comment:
```suggestion```suggestion
        try (PreparedStatement statement = connection.prepareStatement(sql)) {
            statement.execute();
        }
```
```